src/main/java/org/carlmontrobotics/robotcode2023/commands/ArmPeriodic.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.carlmontrobotics.robotcode2023.commands;
+
+import java.util.function.DoubleSupplier;
+
+import org.carlmontrobotics.robotcode2023.Constants;
+import org.carlmontrobotics.robotcode2023.subsystems.Arm;
+
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.*;
+
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ArmPeriodic extends CommandBase {
+  /** Creates a new ArmPeriodic. */
+  private Arm armSubsystem;
+  private DoubleSupplier arm;
+  private DoubleSupplier wrist;
+  private double currArmRad = 0, currWristRad = 0;
+  private double lastTime = 0;
+  private final double EPSILON = 0.0001;
+
+  public ArmPeriodic(Arm armSubsystem, DoubleSupplier arm, DoubleSupplier wrist) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(this.armSubsystem = armSubsystem);
+    this.arm = arm;
+    this.wrist = wrist;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    lastTime = Timer.getFPGATimestamp();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
```

Comment:
Some of what is in this method actually belongs in `Arm.periodic()`. As it stands, goals set by other commands will be ignored. Basically, what you want is for this method to just calculate the goal profile states based on the requested speeds, and then pass those goal profile states to the `Arm` subsystem.  Then `Arm.periodic()` should use whatever the the goal profile states are to create and calculate the profiles and drive the arm/wrist (i.e. the code which does that in this method should be moved to `Arm.periodic()`).